apps/ledger-live-desktop/src/renderer/analytics/segment.ts

```diff
@@ -55,34 +55,22 @@ export function setAnalyticsFeatureFlagMethod(method: typeof analyticsFeatureFla
   analyticsFeatureFlagMethod = method;
 }
 
+let isBatch1Enabled: boolean,
+  isBatch2Enabled: boolean,
+  isBatch3Enabled: boolean,
+  ptxEarnEnabled: boolean;
```

Comment:
it's weird to have a "global" property on ALL events where this only concern one part of our app?
have we considered moving this from a "global" to a "local" approach? (like only adding the metadata on event specific to the feature related to that feature flag 🤔 )

Reply:
this seems like a sensible approach to me in general. However whilst implementing my own AB test analytics, Amine needs more data on the test groups general behaviour in order to get historical data to make sure the ab test groups are unbiased, so requested (at least in my case) that the data be on every event. I think it is AB test specific, and can be removed or targeted once the flag is 100%